pkgs/top-level/all-packages.nix

```diff
@@ -6943,6 +6943,8 @@ with pkgs;
   };
   purePackages = recurseIntoAttrs (callPackage ./pure-packages.nix {});
 
+  conda = callPackage ../development/interpreters/conda { };
```

Comment:
I think we should actually move the expression to `pkgs/tools/package-management/`